papers/templates/papers/paper.html

```diff
@@ -146,6 +146,20 @@ <h1 id="paperTitle" data-params="{csrfmiddlewaretoken:'{{csrf_token}}'}" data-pk
         <div class="detailsContent"><div class="abstract">{% autoescape off %}{{ paper.abstract }}{% endautoescape %}</div></div>
     {% endif %}
 
+    {% with doi=paper.get_doi %}
+        {% if doi and not paper.pdf_url %}
+            <h2>{% trans "Not the author?" %}</h2>
+            <p>
+                {% blocktrans count author_nb=paper.author_count %}
+                    Encourage the author to liberate this publication with the <a href="https://openaccessbutton.org/" target="_blank">Open Access Button</a>.
+                    {% plural %}
+                    Encourage the authors to liberate this publication with the <a href="https://openaccessbutton.org/" target="_blank">Open Access Button</a>.
+                {% endblocktrans %}
+            </p>
+            <p><a href="https://openaccessbutton.org/request?doi={{ doi }}" target="_blank" rel="nofollow" class="btn btn-primary btn-xs"><strong>{% trans "Send request" %}</strong></a></p>
```

Comment:
I'm not 100% convinced by the `btn-xs`, I feel like it lacks a bit of padding. What do you think?